### PR TITLE
NEWS: Updating new for 1.5.0 rc2

### PR DESCRIPTION
**Pull request by shamisp**
- News file update
- Minor clean up of the file
- Copyright updates for 2019

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8559/ for details (Mellanox internal link).

**Comment by shamisp** [comment]
@yosefe ?

File: NEWS [hunk 1]
```diff
@@ -1,23 +1,37 @@
...
+- GPU - Support for stream API and receive side pipelining
+- Malloc hooks using binary instrumentation instead of symbol override
+- Statistics for UCT tag API
+- GPU-to-Infiniband HCA affinity support based on locality/distance (PCIe)
+
+Bugfixes:
+- Multiple bugfixes (full list on github)
+
+Tested configurations:
+- InfiniBand: MLNX_OFED 4.2-4.4, distribution inbox drivers, rdma-core
```
Comment (yosefe) [inline NEWS]: MLNX_OFED 4.4 and 4.5

Comment (shamisp) [reply NEWS]: Ok. I will fix.

**Comment by shamisp** [comment]
@miked-mellanox maybe you can approve ?

**Pull request merged**